rpcs3/rpcs3qt/game_list_frame.cpp

```diff
@@ -698,47 +699,37 @@ void game_list_frame::ShowSpecifiedContextMenu(const QPoint &pos, int row)
 	});
 	connect(removeGame, &QAction::triggered, [=]
 	{
-		if (QMessageBox::question(this, tr("Confirm Delete"), tr("Permanently delete files?")) == QMessageBox::Yes)
+		QMessageBox* mb = new QMessageBox(QMessageBox::Question, tr("Confirm %1 Removal").arg(qstr(currGame.category)), tr("Permanently remove %1 from drive?").arg(qstr(currGame.name)), QMessageBox::Yes | QMessageBox::No, this);
+		mb->setWindowFlags(mb->windowFlags() & ~Qt::WindowContextHelpButtonHint);
```

Comment:
Is `& ~Qt::WindowContextHelpButtonHint` still necessary? You removed all occurences in the commit https://github.com/RPCS3/rpcs3/pull/4159/commits/7e840a9fd750eb0a899553a531e9bc20dd7d12d7

Reply:
weird. I was pretty sure I removed that line during the rebase